Gemfile

```diff
@@ -2,19 +2,21 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
-gem 'rails', '~> 6.0.3', '>= 6.0.3.3'
+gem 'bootsnap', '>= 1.4.2', require: false
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
-gem 'bootsnap', '>= 1.4.2', require: false
+gem 'rails', '~> 6.0.3', '>= 6.0.3.3'
 
 group :development, :test do
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'capybara'
```

Comment:
For what do we use capybara? Do we need it for testing an API?

Reply:
Removed.

Reply:
Yes I can see that, but that was not my question.